themes/cardinal_service/lib/scss/utilities/_cta.scss

```diff
@@ -21,3 +21,13 @@ svg {
     bottom: 12px;
   }
 }
+
+
+.su-link--external {
+  position: relative;
+
+  &::after {
+    position: absolute;
+    top: 10px;
```

Comment:
Does this work for all links with the class? I'm not sure what tags/styles might make the link text size larger

Reply:
Great question! We can target the `su-link` class instead so that the changes are applied any icons that might be used in the future :)

Reply:
I might need to go back to the drawing board for this fix. The `position: absolute` is causing it to overlap any following text as well: 
![image](https://user-images.githubusercontent.com/34019925/94195286-a9ad1280-fe67-11ea-9e81-0c83b003307a.png)

Reply:
you could just add a little margin/padding to the link. i'm ok with the current class target. I'm just not sure if using that class in combination will put the arrow in the right place with the `top: 10px`